karma.config.js

```diff
@@ -0,0 +1,62 @@
+const {
+  NodeModulesPolyfillPlugin,
+} = require('@esbuild-plugins/node-modules-polyfill');
```

Comment:
This is only needed for the `util.inspect` usage in `setup.ts`.

Reply:
Can you add that as a comment in the file?